File: scratch/check_schema.js
const { Pool } = require('pg');
const fs = require('fs');

let connStr = '';
const lines = fs.readFileSync('/var/www/gymdate/.env.local', 'utf8').split('\n');
for (const line of lines) {
  if (line.startsWith('DATABASE_URL=')) {
    connStr = line.split('=')[1].trim().replace(/^['"]|['"]$/g, '');
  }
}

const pool = new Pool({ connectionString: connStr });
pool.query("SELECT column_name, data_type FROM information_schema.columns WHERE table_name = 'partner_requests'")
  .then(res => {
    console.log('PARTNER_REQUESTS COLUMNS:', res.rows);
    pool.end();
  })
  .catch(err => {
    console.error('ERROR:', err);
    pool.end();
  });

File: scratch/check_schema.py
import os
import psycopg2

conn_str = None
with open('/var/www/gymdate/.env.local') as f:
    for line in f:
        if line.startswith('DATABASE_URL='):
            conn_str = line.strip().split('=', 1)[1].strip('\'"')

if conn_str:
    conn = psycopg2.connect(conn_str)
    cur = conn.cursor()
    cur.execute("SELECT column_name, data_type FROM information_schema.columns WHERE table_name = 'partner_requests'")
    cols = cur.fetchall()
    print("partner_requests columns:", cols)
    conn.close()
